docs/source/GetStartedGuide.rst

```diff
@@ -140,9 +140,6 @@ The output might look like this::
 These Python files are considered by Avocado to contain ``INSTRUMENTED``
 tests.
 
-.. These should refer to proper simple tests example but they are currently
-   lacking in our tree. See GitHub issue #628.
-
```

Comment:
Yes, this is obsoleted, but the whole section below does not work anymore as simpletests are located in the same example directory. Would you please update the whole section once you're on it?